third-party/arcanum-worlds/odyssey-of-the-dragonlords/subclass-ranger-amazonian.xml

```diff
@@ -0,0 +1,256 @@
+        </description>
+        <sheet>
+            <description action="Reaction">If you would take damage from an attack or an effect that you can see, your stimfay takes the damage instead of you. Your stimfay must be functional and ready to assist you in order to use this ability, and it must be located within 60 feet of you.</description>
+        </sheet>
+    </element>
+
+    <element name="Pressure Points" type="Archetype Feature" source="Odyssey of the Dragonlords" id="ID_AW_OOTD_ARCHETYPE_FEATURE_RANGER_AMAZONIAN_PRESSURE_POINTS">
+        <description>
+            <p>Starting at 15th level, you know how to attack pressure points in your opponent’s body that will cut off the blood supply to their brain. On your turn, you may use a bonus action to make a special melee attack against one creature. If the attack hits, the creature must make a Constitution saving throw versus your spell save DC. On a failure, the creature is paralyzed for 1 minute or until you use a bonus action to touch them and reverse the effect. At the end of each of the creature’s turns, it repeats the saving throw and the effect ends if it succeeds. You may use this ability a number of times equal to your Wisdom modifier, and you regain all expended uses after a short or long rest.</p>
+        </description>
+        <sheet>
+            <description action="Bonus Action" usage="{{wisdom:modifier}}/Short Rest">Make a melee attack against one creature (using any weapon). If the attack hits, the creature must make a Constitution saving throw versus your spell save DC. On a failure, the creature is paralyzed for 1 minute or until you use a bonus action to touch them and reverse the effect. At the end of each of the creature’s turns, it repeats the saving throw and the effect ends if it succeeds.</description>
+        </sheet>
+    </element>
+
+    <!-- Companion -->
+    <element name="Stimfay" type="Companion" source="Odyssey of the Dragonlords" id="ID_AW_OOTD_COMPANION_RANGER_AMAZONIAN_STIMFAY">
+		<supports>Stimfay Companion</supports>
+		<description>
+			<p>You constructed a stimfay companion to accompany you on your travels.</p>
+		</description>
+		<setters>
+			<set name="strength">10</set>
+			<set name="dexterity">11</set>
+			<set name="constitution">10</set>
+			<set name="intelligence">6</set>
+			<set name="wisdom">8</set>
+			<set name="charisma">7</set>
+            <set name="ac">{{companion:ac}} (natural armor)</set>
+			<set name="hp">{{companion:hp:max}}</set>
+			<set name="speed">10 ft., fly 80 ft.</set>
+			<set name="immunities">fire, poison, psychic</set>
+			<set name="conditionImmunities">charmed, poisoned</set>
+			<set name="saves">Str +{{companion:save}}, Dex +{{companion:save}}, Con +{{companion:save}}, Int +{{companion:intsave}}, Wis +{{companion:wissave}}, Cha +{{companion:chasave}}</set>
```

Comment:
`{{companion:ac}}` are not supported here, just the actual value

same for hp/saves etc